demo-shell-ng2/app/app.component.ts

```diff
@@ -46,7 +46,8 @@ export class AppComponent {
         this.setProvider();
 
         if (translate) {
-            translate.addTranslationFolder();
+            translate.addTranslationFolder('custom', 'custom-translation/');
+            // translate.addTranslationFolder('ng2-alfresco-login', 'custom-translation/alfresco-login');
```

Comment:
We need this code just to show how the developer can override the translation file of a specific component.